.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+**Heads up:** In an effort to unify our design resources, we will disable this repo and migrate Capital Framework's code to [cfpb/design-system](https://github.com/cfpb/design-system) in Summer 2019. Until the migration occurs, please continue to use this repo.
```

Comment:
I'd omit this from the pull request template, since it'll be something we have to remove relatively soon, and in general it would be a good idea to keep the template consistent across projects. If for some reason someone opens a pull request in conflict with the migration, a comment could be left to point them to this notice in the readme.

Reply:
Oh wait, I'd misread the "disable this repo" part. I guess it doesn't really matter in that case.

Reply:
We'll archive it and I think that prevents PRs